velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -192,12 +191,31 @@ TEST_F(NthValueTest, offsetValues) {
 
   std::string overClause = "partition by c0 order by c1";
   std::string offsetError = "Offset must be at least 1";
+  std::string constantKFrameClause = "rows between 0 preceding and current row";
+  std::string columnarKFrameClause =
+      "rows between c2 preceding and current row";
+  std::string kBoundConstantOffsetError =
```

Comment:
notice inconsistency in the naming: kBoundColumnOffsetError vs. columnarKFrameClause

Reply:
Apologies for the inconsistency, these variables are now removed as suggested.